### DISCUSSION TITLE
Antivirus warning upon running debloatware

### DISCUSSION THREAD

**Discussion by Nirajb007** [LeDragoX/Win-Debloat-Tools#155] [General]
Sorry, I got tons of warning when I tried to run this. Is it normal? Sorry I didn't read anywhere that this should be disabled prior to running debloat. I use Bitdefender

**Comment by LeDragoX** [comment]
Can you send a screenshot from the detection? (Use `Win + Shift + S` to select the area to print)
I'm curious of which file is causing this, if you can't, I'll try scanning with this antivirus.

> Is it normal?

No, this isn't supposed to happen, the first time it did detect, was because of other script which wasn't used inside the folders.

**Reply by Nirajb007** [reply]
Hello,
I get alerts when running the script. Bit defender then points towards powershell being executed and then stops any command being executed
